sessions/git-and-github.qmd

```diff
@@ -1 +1,44 @@
-# Review of Git and GitHub {#sec-git-and-github}
\ No newline at end of file
+# Review of Git and GitHub {#sec-git-and-github}
+
+<!-- TODO: Add an introduction here -->
+
+## Learning objectives
+
+This session’s **overall learning outcome** is to enable you to:
+
+1.  Outline the core concepts and functionalities of Git and GitHub and
+    describe their role in facilitating effective collaboration
+
+**Specific objectives** are to:
+
+1.  Explain the concept of formal **version control** and its key
+    benefits
+2.  Describe what a **repository** is in the context of Git and GitHub,
+    including the difference between **local** and **remote**
+    repositories.
+3.  Explain the role of **branching** in managing parallel work
+4.  Explain how **commits** are used to track changes and maintain a
+    history of files.
+5.  Describe how **pull requests** are used to integrate changes and
+    collaborate with team members.
+6.  Describe how GitHub **issues** can be used for managing tasks.
+7.  Explain how Git and GitHub facilitate collaboration by:
```

Comment:
```suggestion
7.  Discuss and explain how Git and GitHub facilitate collaboration by:
```

Just wondering, this might be a good discussion exercise to include at the end of this session to achieve this learning objective.